### DISCUSSION TITLE
Send historical data to Xdrip or Nightscout

### DISCUSSION THREAD

**Discussion by ibnbr** [j-kaltes/Juggluco#169] [Q&A]
I'm using Juggluco with Libre 2, and when I am disconnected from the sensor for a while, Juggluco doesn't send historical data. My chart in Nightscout has a lot of gaps.

Is there any setting to change this? I can`t find this.

**Comment by j-kaltes** [comment]
The glucose broadcasts always contain the current glucose value from Libre 2 sensors. To receive the history values an app needs to use the http://127.0.0.1:17580/x/history command of the web server in Juggluco, see https://www.juggluco.nl/Juggluco/webserver.html
